### ISSUE TITLE
Plan for Python questions

### ISSUE THREAD

**Issue by mageshr2016** [kirkbrunson/ml4t-guide#5]
Hi - Thanks for this nice guide. Really helpful. I'm trying to see whats the best way to add python questions. Currently it is available as text file. Is there any easy way to convert it to a json and add them to the list here?

Thanks again.

**Comment by kirkbrunson** [comment]
@iosnewbie2016 Thanks -- and glad you found this useful!

I don't currently have plans for adding python questions, but would welcome it. The questions would need to be parsed from the text file to json, as well as adding the ability to format the question's code on the front end. Any interest in exploring this?

**Comment by mageshr2016** [comment]
Yes, I am certainly interested to explore. Is there any quick way to
convert these text files to JSON?

**Comment by kirkbrunson** [comment]
I wrote a python script to parse the current ones. Not sure about a quick solution for the python questions since it's been a while since I've seen them. Can you link to file?

**Comment by mageshr2016** [comment]
Sure. Here is the link - https://docs.google.com/document/d/1sVHdIchHNMJro0CbxIBXexCwQ621MPMe-sCG3KTCdEo/pub

If you can share your python script, may be I can try to see if I can add too.

**Comment by kirkbrunson** [comment]
Ahh, I remember this. Started to, but parsing it turned out to be quite a
bit more work than seemed worth it.

I'd either copy/paste (perhaps Fiverr?) or just link to the doc to add
those questions to the site.

Btw, I'm @kirk on slack.

**Comment by ssl9ek** [comment]
https://github.com/kirkbrunson/ml4t-guide/pull/7

Parsing did turn out to be a pain. Here's my solution for a first chunk of python questions.  No special logic for formatting code on front end yet
